css/styles.css

```diff
@@ -215,6 +222,183 @@ body {
   margin-top: -40px;
 }
 
+/************** WORKS SECTION ***************/
+.works-sec {
+  display: grid;
+  grid-template-columns: 20% 20% 20% 20% 20%;
+}
+
+.project {
+  margin-top: 4%;
+  margin-bottom: 4%;
+  grid-column: 2/6;
+  width: 70%;
+  background-color: #fff;
+  border-radius: 1%;
+}
+
+.img-prg {
+  width: 40vh;
+  height: 40vh;
+  margin-top: 20px;
+  margin-left: 20px;
+  margin-bottom: 20px;
+}
+
+.project-info {
+  width: 50%;
+  margin: auto;
+}
+
+.project-title {
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 32px;
+  width: 60%;
+}
+
+.project-det {
+  font-size: 13px;
+  justify-content: space-between;
+  width: 50%;
+}
+
+.project-det-desc,
+.project-det-yr {
+  color: #7a869a;
+}
+
+.prgt-det-circl {
+  background-image: url('../assets/prgt-circle-det.svg');
+  margin-top: 1%;
+  background-repeat: no-repeat;
+  width: 10px;
+}
+
+.project-desc {
+  font-size: 15px;
+  color: #344563;
+}
+
+.project-tags {
+  justify-content: space-between;
+  width: 30%;
+}
+
+.project-tag {
+  background-color: #ebebff;
+  color: #6070ff;
+  font-size: 12px;
+  font-weight: 500;
+  padding: 2%;
+  border-radius: 10%;
+}
+
+.project-link {
+  margin-top: 2%;
+  margin-bottom: 4%;
+  background-color: transparent;
+  padding: 2%;
+  font-size: 17px;
+  border-radius: 4%;
+  color: #396df2;
+  border-color: #396df2;
+}
+
+/************** ABOUT SECTION ***************/
+
+.about-me {
+  background-color: #fff;
+  border-top-right-radius: 200px;
+}
+
+.about-container {
+  width: 60%;
+  margin: auto;
+  padding: 4%;
+}
+
+.about-section-one {
+  width: 40%;
+}
+
+.languages {
+  width: 50%;
+  padding: 4%;
+}
+
+.socials-about {
+  justify-content: space-between;
+  width: 60%;
+}
+
+.about-title {
+  color: #091e42;
+  font-size: 40px;
+  width: 50%;
+}
+
+.about-desc {
+  color: #091e42;
+  font-size: 16px;
+}
+
+.languages-content {
+  display: none;
+}
+
+.collapse-btn {
+  padding: 5%;
+  margin-top: 4px;
+  border-radius: 0;
+  border-top: 0;
+  border-left: 0;
+  border-right: 0;
+  border-color: #dfe1e6;
+  background-color: #fff;
+  font-size: 20px;
+  font-weight: 500;
+  text-align: left;
+}
+
+.collapse-btn::after {
+  content: '\003E';
+  color: #acb7c3;
+  float: right;
+}
+
+.active::after {
+  content: '\003E';
+  color: #acb7c3;
+  -webkit-transform: rotate(90deg);
+  transform: rotate(90deg);
+}
+
+.langs-container {
+  padding: 3%;
+  justify-content: space-between;
+  width: 70%;
+}
+
+.language-item {
+  background-color: #f7f7f9;
+  width: 20%;
+  padding: 4%;
+  border-radius: 5px;
+  flex-direction: column;
+}
+
+.language-text {
+  color: #253858;
+  font-weight: 500;
+  font-size: 15px;
+  padding-top: 10px;
+}
+
+.img-lang {
+  width: 50px;
+}
+
 @media only screen and (max-width: 1023px) {
```

Comment:
Are media queries prohibited, and was it specified in the instructions? And I thought we still use media queries in mobile first design

Reply:
This is a mobile-first application. By that definition, a media query is not necessary, **YET**, if your app is built as a mobile-first application. It should not matter what your application looks like in the desktop version as long as your mobile screen is taken care of. The desktop version will be treated sometime later in the module. It should not be a concern now.

I had wanted to mark it as optional but somehow, in your project, some parts of your project which should appear on the mobile screen only seem to appear on the desktop screen probably because of the media query. For example, the **big right border-radius** on top of ABOUT MYSELF was supposed to appear on mobile but it only appears in the desktop version.

Reply:
@TedLivist, I tried building as the template that we were given... considering we had to build a hamburger button when the screen shrunk, I went on to use media queries

Reply:
I'll try to find a solution based on the feedback given, I'll try to submit ASAP

Reply:
@Ntare22, okay. That probably should not have bothered you. You should build with HTML and CSS strictly, using the template given.

Yeah sure. No problem. You can also ignore the JavaScript implementation. As you could see it is not a requirement for this particular project. Don't waste time on JavaScript